packages/core/src/components/link-button/link-button.scss

```diff
@@ -27,31 +28,21 @@
     cursor: pointer;
     text-decoration: none;
 
-    &:hover {
+    @include hover {
       color: var(--theme-color-dynamic--hover);
 
       .underline {
         text-decoration: underline;
         text-underline-offset: 0.2rem;
-
-        &.disabled {
-          color: var(--theme-color-weak-text);
-          text-decoration: none;
-        }
       }
     }
 
-    &:active {
+    @include active {
```

Comment:
The `&` selector in combination with `:host` on the first level will not work. I created a ghist regarding major changes between scoped and shadow https://gist.github.com/danielleroux/8812ce6ec2ea34bc388a2cd53ac1e649